tests/tracing/helper.py

```diff
@@ -145,6 +152,7 @@ def reset_autolog_state():
         # 1. Remove post-import hooks (registered by global mlflow.autolog() function)
         mlflow.utils.import_hooks._post_import_hooks.pop(flavor, None)
 
+    for flavor in AUTOLOGGING_INTEGRATIONS.keys():
```

Comment:
note: Small optimization for the clean up fixture. We don't need to call `autolog(disable=True)` when it was not enabled during the test i.e., `AUTOLOGGING_INTEGRATIONS` does not include the flavor name.